File: Java/Node/src/Node/Biz/Interfaces/GetServices/IPostProcess.java
package Node.Biz.Interfaces.GetServices;

import java.rmi.RemoteException;

import Node.Biz.Custom.PostParam;
/**
 * <p>This class create GetServices Post Process Interface.</p>
 * <p>Company: enfoTech & Consulting, Inc.</p>
 * @author enfoTech
 * @version 2.0
 */


public interface IPostProcess {
	/**
	 * Execute
	 * @param token The authentication token
	 * @param serviceType The service type name
	 * @param param PostParam object
	 * @throws RemoteException
	 * @return PostParam object
	 */
  public PostParam Execute (String token, String serviceType, PostParam param) throws RemoteException;
}

File: EN Node for .NET environment/Node.Core/Biz/Interfaces/GetServices/IPostProcess.cs
using System;

using Node.Core.Biz.Manageable.Parameters;
using Node.Core.Biz.Objects;

namespace Node.Core.Biz.Interfaces.GetServices
{
    /// <summary>
    /// The Interface for GetServices PostProcess
    /// </summary>
    public interface IPostProcess
    {
        /// <summary>
        /// PostProcess for GetServices.
        /// </summary>
        /// <param name="token">The Token which is result of Authentication Operation.</param>
        /// <param name="serviceType">Specifies which type of service are to be used.</param>
        /// <param name="param">Operation related parameters</param>
        /// <returns>A list of Opeation Name</returns>
        void Execute(string token, string serviceType, PostParam param);
    }
}
